docs: swatting various bugs (#117)
neo4j/docs-kafka-connector: modules/ROOT/pages/sink.adoc

Use a Cypher-like pattern syntax to transform incoming Kafka messages into Cypher write queries.
xref:sink/cud.adoc[Create Update Delete (CUD) File]::
Transform incoming CUD-formatted messages into Cypher write queries.

> Not sure which one fits your use case? See xref:sink/best-practices.adoc#choosing-a-strategy[Choosing a sink strategy].

Reviewer: ```suggestion
See xref:sink/best-practices.adoc#choosing-a-strategy[Choosing a sink strategy] for advice on choosing the best strategy for your use case.
```
